interpreters/duktape/Makefile

```diff
@@ -48,23 +48,30 @@ CFLAGS += -I$(DUKTAPE_UNPACK)/extras/print-alert
 CFLAGS += -DDUK_CMDLINE_CONSOLE_SUPPORT
 CFLAGS += -DDUK_CMDLINE_PRINTALERT_SUPPORT
 
+APPINC = $(CURDIR)/$(DUKTAPE_UNPACK)
+
 $(DUKTAPE_TARBALL):
-	@echo "Downloading $(DUKTAPE_TARBALL)"
-	@wget $(DUKTAPE_URL)
+	$(Q) echo "Downloading $(DUKTAPE_TARBALL)"
+	$(Q) wget $(DUKTAPE_URL)
 
 $(DUKTAPE_UNPACK): $(DUKTAPE_TARBALL)
-	@echo "Unpacking $(DUKTAPE_TARBALL) to $(DUKTAPE_UNPACK)"
-	@tar xvfJ $(DUKTAPE_TARBALL)
-	@echo "Patching $(DUKTAPE_UNPACK)"
-	@patch -p0 < duk_cmdline.patch
-	@touch $(DUKTAPE_UNPACK)/.patch
+	$(Q) echo "Unpacking $(DUKTAPE_TARBALL) to $(DUKTAPE_UNPACK)"
+	$(Q) tar xvfJ $(DUKTAPE_TARBALL)
+	$(Q) mv duktape-$(DUKTAPE_VERSION) $(DUKTAPE_UNPACK)
+	$(Q) ln -sf $(APPINC)/src-noline/duktape.h $(APPINC)/duktape.h
```

Comment:
change all $(APPINC) to $(DUKTAPE_UNPACK)?

Reply:
$(DUKTAPE_UNPACK) is a relative path, may cause some problem.